pkg/client/src/app/queries/applications.ts

```diff
@@ -15,14 +15,17 @@ export const useFetchApplications = (
   defaultIsFetching: boolean = false
 ): IFetchState => {
   const [applications, setApplications] = useState<Application[]>([]);
-  const { isLoading, error, refetch } = useQuery("applications", () =>
-    getApplications()
-      .then(({ data }) => {
-        setApplications(data);
-      })
-      .catch((error) => {
-        console.log("error, ", error);
-      })
+  const { isLoading, error, refetch } = useQuery(
+    "applications",
+    () =>
+      getApplications()
+        .then(({ data }) => {
+          setApplications(data);
```

Comment:
Take a look at using the queryClient here instead of react state.  

[query client link ](https://react-query.tanstack.com/reference/QueryClientProvider)const queryClient = useQueryClient()

Reply:
This is actually a mistake. That change has nothing to do with this patch.
Although that's something to discuss separately.